### PR TITLE
init: also retry starting units

### PR DESCRIPTION
**Pull request by chr0n1x**
Someone kill me

### CODE REVIEW

**Comment by ts3ng** [comment]
👍  thanks for fixing and apologies for the noise.

File: init [hunk 1]
```diff
@@ -89,15 +89,19 @@ function submit-fleet-unit() {
...
     submit-fleet-unit ${SCRIPTDIR}/$VERSION/fleet-local/${NODE_ROLE}/${unit}
-    sudo fleetctl start "${unit%.service}${COREOS_PRIVATE_IPV4}"
+    start-fleet-unit "${unit%.service}${COREOS_PRIVATE_IPV4}"
 done
 # GENERAL-PURPOSE UNITS
 for unit in $(ls ${SCRIPTDIR}/$VERSION/fleet|grep -e '.service$\|.timer$')
 do
     submit-fleet-unit ${SCRIPTDIR}/$VERSION/fleet/${unit}
-    sudo fleetctl start ${SCRIPTDIR}/$VERSION/fleet/${unit}
+    start-fleet-unit ${SCRIPTDIR}/$VERSION/fleet/${unit}
```
Comment (adobejmong) [inline init:106]: Just to be consistent with line 100, can you also pass this in as:

```
start-fleet-unit "${SCRIPTDIR}/$VERSION/fleet/${unit}"
```

**Comment by anshupande** [comment]
👍

**Comment by chr0n1x** [comment]
@ts3ng I mean kill me because I'm dumb, no ones' fault but mine haha

**Pull request closed without merging**